Doctrine/EventSubscriber.php

```diff
@@ -64,7 +66,12 @@ public function persistObject(DataEvent $event)
         }
 
         $objectManager->persist($event->getData());
-        $objectManager->flush();
+        try {
+            $objectManager->flush();
+        } catch (\Exception $e) {
```

Comment:
No cause I throw the exception at the end of my catch: https://goo.gl/apbyaL
PHP `finally` block does not allow to re-throw an exception